wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusProvider.swift

```diff
@@ -0,0 +1,124 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+// sourcery: AutoMockable
+public protocol MLSConversationVerificationStatusProviderInterface {
```

Comment:
To me a "status provider" sounds like something that I can get a status from. It's weird to see a single method `updateStatus` declared.

Reply:
Agree. I'll do the refactoring as part of this ticket: https://wearezeta.atlassian.net/browse/WPB-6486